packages/scaffold-ui/src/partials/w3m-all-wallets-list/index.ts

```diff
@@ -53,30 +53,31 @@ export class W3mAllWalletsList extends LitElement {
   public override render() {
     return html`
       <wui-grid
-        data-scroll=${!this.initial}
+        data-scroll=${!this.loading}
         .padding=${['0', 's', 's', 's'] as const}
         columnGap="xxs"
         rowGap="l"
         justifyContent="space-between"
       >
-        ${this.initial ? this.shimmerTemplate(16) : this.walletsTemplate()}
+        ${this.loading ? this.shimmerTemplate(16) : this.walletsTemplate()}
         ${this.paginationLoaderTemplate()}
       </wui-grid>
     `
   }
 
   // Private Methods ------------------------------------- //
   private async initialFetch() {
+    this.loading = true
     const gridEl = this.shadowRoot?.querySelector('wui-grid')
-    if (this.initial && gridEl) {
+    if (gridEl) {
       await ApiController.fetchWallets({ page: 1 })
       await gridEl.animate([{ opacity: 1 }, { opacity: 0 }], {
         duration: 200,
         fill: 'forwards',
         easing: 'ease'
       }).finished
-      this.initial = false
-      gridEl.animate([{ opacity: 0 }, { opacity: 1 }], {
+      this.loading = false
+      gridEl.animate?.([{ opacity: 0 }, { opacity: 1 }], {
```

Comment:
This is interesting, in which cases the `animate` function doesn't exist? Maybe some old browsers?